articles/Introduction-to-time-series/index.md

```diff
@@ -0,0 +1,64 @@
+
+#### Time-series Analysis
+Time series analysis can be defined as getting meaningful insight and characteristics to comprehend it better. Time series is beneficial in making predictions. It understands the scope of nature and uses it for future forecasting.
+
+#### Time Series Forecasting methods 
+Now let’s look at four time-series forecasting methods. To understand them, let's have a brief introduction on Autoregression(AR) and Moving average models. An AR model simply uses a linear combination of the target's past values to make forecasts, while a moving-average model is an approach for modeling univariate time series. The moving-average model specifies that the output variable relies linearly on a stochastic term's current and various past values.
+
+1. Autoregressive Integrated Moving Average (ARIMA): This method combines both the Autoregression and Moving Average model. In ARIMA models, the steps are in sequence as a linear function on the difference in observation and residual errors at previous steps. This method works best for univariate time series with trend and without seasonal components.
+2. Seasonal Autoregressive Integrated Moving-Average(SARIMA): In the SARIMA model, the next steps are in a linear function style of the different observations, errors, differenced seasonal observation, and seasonal errors at the previous time steps. The SARIMA is fitted for univariate time series with the trend or seasonal components.
+3. Vector Autoregression (VAR): This method models the next step in each of the time series using the AR methods. It is the generalization of AR to multiple parallel time series e,g multivariate time series.
+4. Simple Exponential smoothing(SES): In this method, the model uses the next step as an exponentially weighted linear function of observation at previous time steps. This method works best for univariate time series without trend and seasonal components.
+
+#### Types of Time series
+We are going to look at two types of time series:
+
+**Deterministic time series:** A deterministic time series is expressed by an analytic expression. It doesn’t have random or probabilistic aspects. In deterministic time series, the past and future are explicitly specified by these derivative values at that time.
+
+**Non-deterministic time series:** This type of time-series means it cannot be described by an analytic expression and has some random aspect making it unable to describe the behavior. They are two reasons that make a time series non-deterministic. First is if all the information required to describe it is not available. Secondly, if the nature of the process for generation is inherent.
```

Comment:
What do you mean by nature of process for generation is inherent?

Reply:
Explain the meaning of process is inherent.

Reply:
It means that non-deterministic time series  have a random aspect which makes their process of generating to be random too(because of its inherently random nature)